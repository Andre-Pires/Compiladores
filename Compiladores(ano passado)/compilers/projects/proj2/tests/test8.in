int exp4(int x){
  return x * (x * (x * (x * 1.2)));
}
