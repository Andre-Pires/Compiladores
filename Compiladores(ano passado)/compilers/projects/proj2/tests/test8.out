warning: type cast int to float, line 2
type error: int expression expected, line 2
